js/main.js

```diff
@@ -98,18 +107,28 @@ $bottomSheetHeader.addEventListener('click', function (event) {
     $bottomSheet.classList.add('minimized');
     $bottomSheet.classList.add('no-scroll');
     $bottomSheet.classList.remove('inner-scroll');
-    $bottomSheetCloseButton.classList.add('invisible');
+    $bottomSheetButtons.classList.add('invisible');
     $bottomSheetExpandButton.textContent = 'expand_less';
     $mainAppArea.classList.remove('no-scroll');
     $mainAppArea.classList.add('inner-scroll');
+    toggleDeleteMode(false);
+  } else if (event.target.tagName === 'SPAN' && ['delete_forever'].includes(event.target.textContent)) {
+    // turn on delete mode
+    clearInterval(deleteModeInfoBoxTimerId);
```

Comment:
I'm almost certain that clearInterval and clearTimeout do the same thing, but it's confusing to use clearInterval when your intent is to clear a timeout.

Reply:
That was my mistake, I was probably thinking of intervals when I was writing that...